src/lib/api/builder.ts

```diff
@@ -704,6 +704,10 @@ export class BuilderAPI extends BaseAPI {
     return this.request('get', '/thirdParties', { manager })
   }
 
+  fetchThirdPartyAvailableSlots(thirdPartyId: string): Promise<number> {
+    return this.request('get', `/thirdParties/${thirdPartyId}/slots`)
```

Comment:
Question! I'm actually not sure. Does this return an actual number? meaning isn't it a string?

Reply:
I believe it's coming as a `number` from the request:

![image](https://user-images.githubusercontent.com/8763687/153576562-5ecdf518-e4d6-4e11-bd8d-7e2c8b1d2aaa.png)

Reply:
Awesome then!